### ISSUE TITLE
Use together with other Xunit.IRunnerReporter?

### ISSUE THREAD

**Issue by AutioL25187-pki** [reportportal/agent-net-xunit#13]
We have implemented our own Xunit.IRunnerReporter that writes log to files. But seems that we can't use it together with ReportPortal.XUnit NuGet. I tested and there can't be two implementations of Xunit.IRunnerReporter. Xunit will use the first one in alphabetic order.

How can we use ReportPortal.XUnit NuGet and custom Xunit.IRunnerReporter implementation together?

**Comment by nvborisenko** [comment]
Yeah, it's weird. Xunit really use only single reporter by design. It's weird situation because I proposed PR https://github.com/xunit/xunit/pull/1863 to support multiple reporters, but there is no progress.

As a good workaround I can propose to implement your custom kind of `MulticastRunnerReporter`, which will be picked up by xunit runner. And then this reporter will transfer all messages to the parties (your custom logger and our reporter).
